src/components/ProgressIndicator.css

```diff
@@ -12,11 +12,17 @@
     flex-grow: 1;
     height: 0.5rem;
     border-radius: 0.25rem;
-    background: #e5e5e5;
+    background: var(--nimiq-blue);
     margin: 0.5rem;
+    opacity: 0.1;
 }
 
-.nq-card-header .progress-indicator ~ * {
+.nq-blue-bg .progress-indicator .indicator {
+    background: white;
+    opacity: 0.25;
+}
+
+.nq-card-header .progress-indicator + * {
```

Comment:
Then I need to make a second rule for the arrow, because we now also have `.nq-notice`s in the header, which are also affected by a sibling selector.